compiler/src/iree/compiler/ExternalInterfaces/UtilExternalModels.cpp

```diff
@@ -200,10 +200,12 @@ struct HoistableLinalgOpInterface
     : public IREE::Util::HoistableOpInterface::ExternalModel<
           HoistableLinalgOpInterface<OpTy>, OpTy> {
   bool isHoistableOp(Operation *) const { return true; }
+  /// FillOp and broadcasting ops are not hoistableLeaf ops, since it is
```

Comment:
style nit: adding a blank line before the comment usually helps the readability.

see https://google.github.io/styleguide/cppguide.html#Vertical_Whitespace for more details.